usaspending_api/reporting/models.py

```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
Other apps have their own packages for models. Do we do this once we reach a certain number of models in that app?

Reply:
models are separated into individual files after the models grow enough to make a single file too long